dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -1417,6 +1500,8 @@ components:
             $ref: '#/components/schemas/FileFormat'
           type: array
           uniqueItems: true
+        parent:
+          $ref: '#/components/schemas/EntryObjectObject'
```

Comment:
Where'd these changes come from?

Reply:
I saw them start when merging the hotfix with develop. 
Might be some interaction between the new parentid column and some changes we did to openapi on develop